packages/hardhat-core/src/internal/core/runtime-environment.ts

```diff
@@ -50,6 +51,10 @@ export class Environment implements HardhatRuntimeEnvironment {
 
   public entryTaskProfile?: TaskProfile;
 
+  public version: string | null;
```

Comment:
Why don't you just do `public version: getHardhatVersion()`?

Reply:
Sorry @fvictorio  not able to understand how the return type will be function

Reply:
Yeah, sorry, I meant `public version: string = getHardhatVersion()`.

This made me realize that it doesn't make sense for that function to return `string | null`, so I updated that. I also changed the builtin `help` task to use this new field instead of reading it from the package.json.